src/Function.cpp

```diff
@@ -485,6 +498,7 @@ Function::make_random_signature(const CGContext& cg_context, const Type* type, c
 
 	// dummy variable representing return variable, we don't care about the type, so use 0
 	string rvname = f->name + "_" + "rv";
+	f->alias_name = f->name + "_alias";
```

Comment:
We should make the alias name randomized.

Reply:
The alias names are generated randomized. To distinguish between original name and alias name, I am using "_alias".

For example - random function name is generated as func_51(), hence its alias name is func_51_alias. Is this fine to you?